Gem/Info.plist

```diff
@@ -1,36 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-	<dict>
-		<key>CFBundleURLTypes</key>
-		<array>
-			<dict>
-				<key>CFBundleTypeRole</key>
-				<string>Viewer</string>
-				<key>CFBundleURLName</key>
-				<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-				<key>CFBundleURLSchemes</key>
-				<array>
-					<string>gem</string>
-				</array>
-			</dict>
-			<dict>
-				<key>CFBundleTypeRole</key>
-				<string>Viewer</string>
-				<key>CFBundleURLName</key>
-				<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-				<key>CFBundleURLSchemes</key>
-				<array>
-					<string>gemwallet</string>
-				</array>
-			</dict>
-
-		</array>
-		<key>ITSAppUsesNonExemptEncryption</key>
-		<false/>
-		<key>NSCameraUsageDescription</key>
-		<string>Gem Wallet needs access to your Camera to scan QR Codes.</string>
-		<key>NSFaceIDUsageDescription</key>
-		<string>Enabling Face ID helps Gem Wallet keep your assets secure.</string>
-	</dict>
+<dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+			<key>CFBundleURLName</key>
+			<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>gem</string>
+			</array>
+		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+			<key>CFBundleURLName</key>
+			<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>gemwallet</string>
+			</array>
+		</dict>
+	</array>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
+</dict>
```

Comment:
`NSCameraUsageDescription` and `NSFaceIDUsageDescription` has been removed?

Reply:
info.plist not in use for permissions. 
<img width="985" alt="image" src="https://github.com/user-attachments/assets/ae8d6e2b-62cf-4100-97b6-0a4a18556e1d">
Targets->Gem->Info we have the usage descriptions there